crherman7/rechunk
# feat(packages): ts-config | eslint | prettier

## Scope

[closes issue-#40](https://github.com/crherman7/rechunk/issues/40)

## Implementation

* created eslint shared configs
* created prettier shared configs
* created tsconfig shared configs

## Screenshots

<details>
  <summary>ios</summary>

  https://github.com/user-attachments/assets/7bf216e7-92c3-4818-b815-79687f2b2a8e
</details>

<details>
  <summary>android</summary>
  
  https://github.com/user-attachments/assets/8277f073-a306-4832-a15b-8275b4389a60
</details>


## How to Test

* go to any example and run `lint` script
* or run `pnpm eslint --print-config ${path to file}` to see `eslint` config

## Emoji Guide

**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**

E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.

> 🟢 Nice refactor!

> 🟡 Why was the default value removed?

E.g: Blocking feedback must be addressed before merging.

> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | RED |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |
